airbyte-cdk/python/setup.py

```diff
@@ -66,7 +66,7 @@
     packages=find_packages(exclude=("unit_tests",)),
     install_requires=[
         "backoff",
-        "jsonschema==2.6.0",
+        "jsonschema==3.2.0",
```

Comment:
we need to verify somehow that it is compatible with our current code (but I don't know how)

Reply:
yes, but is the standard lib and unit tests have checked all cases with schemas.
2.6.0 is very old version and I updated it because there were warnings about the future incompatibility
I hope to the Airbyte team will validate this.